src/clients/aws_log_group_client.py

```diff
@@ -39,7 +39,7 @@ def log_group_enforcement_actions(
                 actions.append(PutLogGroupSubscriptionFilterAction(logs=self.logs, log_group_config=log_group_config))
             if log_group.retention_days != log_group_config.logs_group_retention_policy_days:
                 actions.append(PutLogGroupRetentionPolicyAction(logs=self.logs, log_group_config=log_group_config))
-            if not set(PLATSEC_SCANNER_TAGS).issubset(log_group.tags):
+            if not skip_tags and not set(PLATSEC_SCANNER_TAGS).issubset(log_group.tags):
```

Comment:
```suggestion
            if not (skip_tags and set(PLATSEC_SCANNER_TAGS).issubset(log_group.tags)):
```
alternative if perfered